_posts/2017-3-26-armory-and-hard-forks.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Armory and Hard Forks
+category: news
+date: 2017-03-26 23:30:00 -0500
+---
+
+The Armory developers (goatpig, droark, and achow101) oppose hard forks that may attack the original chain. The Armory developers support Segregated Witness and Bitcoin Core. Should a long-lasting hard fork that does not attack the original chain exist, we would consider implementing functionality required to allow Armory users to transact on that chain only if absolutely necessary (e.g. the transaction format for that chain is changed). Depending on how drastic the changes are, we may implement the change fully, implement a migration tool to another wallet that supports the change, or simply not support the fork at all.
+
+Although the Armory developers do not support such hard forks, the Armory wallet software does not perform any consensus checks as it relies on its connection to a local Bitcoin node. This node is typically Bitcoin Core, but anything based upon that will work as well; this includes Bitcoin Unlimited. Thus should a hard fork occur, so long as no changes are made to the transaction or block format, Armory will be compatible with the hard fork and will allow users to continue to transact on the forked network.
```

Comment:
"This node is typically Bitcoin Core, but anything based upon that will work as well; this includes Bitcoin Unlimited. Thus should a hard fork occur, so long as no changes are made to the transaction or block format, Armory will be compatible with the hard fork and will allow users to continue to transact on the forked network."

This node is typically Bitcoin Core, but any node based upon Core, including Bitcoin Unlimited, will also work so long as no changes are made to the transaction or block formats. So long as both formats do not change, Armory should be compatible with the hard fork and all users to continue to transact on the forked network.

(**NB**: I say "should" in order to be safe. If an Unlimited-style fork occurs, I'm pretty sure Armory will be fine. It's also theoretically possible that another hard fork down the road could leave the formats alone and still be incompatible somehow.)